### ISSUE TITLE
Configuration file in different directory won't work as expected

### ISSUE THREAD

**Issue by flipace** [Codeception/CodeceptJS#282]
Hi, first - thanks for this awesome lib.

We're using [builder](https://github.com/FormidableLabs/builder) for our projects and would like to include codeceptjs in our archetype(s).

This means we'd like to put the configuration file into a folder like ```node_modules/org/archetype/config/codeceptjs/config.js```.

As far as I can see, there's no option to specify the configuration file via the command line yet (as for example with webpack --config=....)

I tried to do

```codeceptjs run node_modules/org/archetype/config/codeceptjs/``` but it won't run the tests.

It only outputs  "OK | 0 passed".

Do I have to do anything different?

**Command**:
```shell
flipace@mbpro:~/Development/project$ codeceptjs run node_modules/org/builder-archetype/config/integration-tests
CodeceptJS v0.4.8
Using test root "/Users/flipace/Development/project/node_modules/org/builder-archetype/config/integration-tests"


  OK  | 0 passed   // 0ms
```

**Config**:
```Javascript
exports.config = {
  tests: path.join(ROOT, 'src/integration-tests/build/tests.bundle.js'),
  timeout: 10000,
  output: path.join(ROOT, 'reports/integration-tests'),
  helpers: {
    WebDriverIO: {
      url: process.env.URL || 'http://localhost:3000',
      user: process.env.BROWSERSTACK_USER || 'none',
      key: process.env.BROWSERSTACK_KEY || 'key',
      browser: browser.browserName,
      desiredCapabilities: Object.assign({}, browser, {
        'browserstack.local': process.env.BROWSERSTACK_LOCAL || false
      })
    }
  },
  include: {
    I: path.join(ROOT, 'src/integration-tests/custom-steps.js')
  },
  bootstrap: false,
  mocha: {},
  name: 'someName'
};
```

**Comment by DavertMik** [comment]
Please try new `--config` option implemented by @cnworks . Available since 0.4.9

**Comment by flipace** [comment]
@DavertMik i think you have to publish it to npm ;)

**Comment by DavertMik** [comment]
Yep, I need to test it locally and include some more fixes before that

14 лист. 2016 р. 17:16 "Patrick Neschkudla" <notifications@github.com> пише:

**Comment by flipace** [comment]
Alright, I just had a chance to test the new --config command!

At first, I still experienced the same issue, except that the "Using test root" message displayed the current working directory (which seemed like an improvement :)).

After fiddling around I figured out what caused my issue - I tried to use an absolute path for "tests" and this led to the weird behavior, when I changed it to a relative path (from the root of the project) it finally worked!
